service/src/main/java/org/cbioportal/service/impl/SampleServiceImpl.java

```diff
@@ -174,31 +174,51 @@ public BaseMeta fetchMetaSamples(List<String> sampleListIds) {
         return sampleRepository.getSamplesByInternalIds(internalIds);
     }
 
-    private void processSamples(List<Sample> samples, String projection) {
+    private Boolean isProfiledWithSV(MolecularProfile p) {
+        return (p.getMolecularAlterationType().equals(MolecularProfile.MolecularAlterationType.STRUCTURAL_VARIANT) ||
+                hasFusionsAsMutations(p));
+    }
 
+    private Boolean hasFusionsAsMutations(MolecularProfile p) {
```

Comment:
We should add the usual:
TODO: remove this logic once all the fusions are migrated to structural variants